pwiz_tools/Skyline/TestData/CommandLineImportTest.cs

```diff
@@ -215,32 +218,34 @@ public void ConsoleImportPeptideListTest()
             var docPath = TestFilesDir.GetTestPath("blank.sky");
             var outPath = TestFilesDir.GetTestPath("import-fasta-file.sky");
             var fastaPath = TestFilesDir.GetTestPath("bov-5-prot.fasta");
+            const int expectedProtCount = 5;
+            const int expectedPepCount = 40;
             var listPath = TestFilesDir.GetTestPath("peplist.txt");
             var listPath3 = TestFilesDir.GetTestPath("peplist3.txt");
             var listsPath = TestFilesDir.GetTestPath("peplists.txt");
 
             // Create initial document for comparisons by importing a FASTA file
-            var output = RunCommand("--in=" + docPath,
-                "--out=" + outPath,
-                "--import-fasta=" + fastaPath);
+            var output = RunCommand(CommandArgs.ARG_IN + docPath,

```

Comment:
It is something we used to do a lot at LabKey. It is easier to type things out and know you got them correct because of statement completion. What did Matt call his new argument? Much easier to hunt through the list of CommandArgs.ARG_ than to have to go search the code. I felt this also made it less necessary for me to copy-paste everything as I created new cases.